advanced/editor-command-identifiers.md

```diff
@@ -402,4 +424,92 @@ tinymce.execCommand('mceToggleEditor', false, '<editor_id>');
 | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | mceAddEditor    | Converts the specified HTML or DOM element into an editor instance with the specified ID.                                                                                                                                                                               |
 | mceRemoveEditor | Removes an editor instance with the specified ID.                                                                                                                                                                                                                       |
-| mceToggleEditor | Runs mceAddEditor if an editor is not detected for the specified ID, otherwise it runs either [hide]({{ site.baseurl }}/api/tinymce/tinymce.editor/#hide) if the editor is visible or [show]({{ site.baseurl }}/api/tinymce/tinymce.editor/#show) if it is not visible. |
\ No newline at end of file
+| mceToggleEditor | Runs mceAddEditor if an editor is not detected for the specified ID, otherwise it runs either [hide]({{ site.baseurl }}/api/tinymce/tinymce.editor/#hide) if the editor is visible or [show]({{ site.baseurl }}/api/tinymce/tinymce.editor/#show) if it is not visible. |
+
+## Queryable command states
+
+{{site.productname}} provides the `queryCommandState` API to allow developers to determine the current state of selected content. The query will return `true` if the content is formatted using the same CSS styles and elements used by the corresponding command.
+
+### Listing core and plugin queryable command states
+
+To retrieve a list of avaliable queryable states from the active editor, run the following command from the browser console:
+
+```js
+tinymce.activeEditor.editorCommands.commands.state;
+```
+
+### Available queryable command states
```

Comment:
The case of many of these commands are incorrect (technically doesn't matter though as the lookup is case insensitive). Anyways, it should be the same as the `execCommand` name, which for most of these are title case with the exception of ones starting with "mce", as they are camel case.

You can see the correct case in the source as well if that helps: https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/api/EditorCommands.ts#L542

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1590/commits/c4f6e844ae7168e5343fd3e281d776e5355b09ac